pkg/scheduler/scheduler.go

```diff
@@ -367,8 +367,9 @@ func (sched *Scheduler) assumeVolumes(assumed *v1.Pod, host string) (allBound bo
 			invalidPredicates := sets.NewString(predicates.CheckVolumeBindingPred)
 			sched.config.Ecache.InvalidatePredicates(invalidPredicates)
 		}
+		return allBound, err
 	}
-	return
+	return true, nil
```

Comment:
this seems odd to me.
the original code looks fine, unless there is a catch here.

Reply:
when set `--feature-gates=VolumeScheduling=False` , `assumeVolumes` should return true to skip func `bindVolumes`. now it retrun false(default), cause `bindVolumes` panic
#70146